scripts/globals/pets/wyvern.lua

```diff
@@ -148,8 +152,15 @@ end;
 -- onMobDespawn Action
 -----------------------------------
 
-function onMobDeath(mob)
+function onMobDeath(mob, player)
     local master = mob:getMaster();
+    local pet = player:getPet();
+    local numLvls = pet:getLocalVar("level_Ups");
+    if (numLvls ~= nil) then
```

Comment:
`CLuBaseEntity:getLocalVar` returns 0 which i dont think is equivalent to `nil` in lua  (see: http://lua-users.org/wiki/ExpressionsTutorial)
```lua
if numLvls ~= 0 then
```
should do it